src/main/java/me/jellysquid/mods/sodium/client/render/chunk/ChunkRenderManager.java

```diff
@@ -103,6 +103,7 @@ public ChunkRenderManager(SodiumWorldRenderer renderer, ChunkRenderBackend<T> ba
         this.world = world;
 
         this.builder = new ChunkBuilder<>(backend.getVertexType(), this.backend);
+        this.builder.SetupWithRender(renderDistance); // Split out for compat...
```

Comment:
This is split out because other mods love injecting into Sodium. Ideally we would just pass this into the constructor.